ocaml/forkexecd/helper/vfork_helper.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ */
+
+#ifndef GUARD_hRiZStWEvmjkFidRuwMC9c
+#define GUARD_hRiZStWEvmjkFidRuwMC9c
+
+#pragma once
+
+// Common structure to pass errors from helper to library
+typedef struct {
+    // numeric C error
+    int err;
+    // message
+    char msg_buf[1000];
+} msg_t;
+
+#endif
+
+// vim: expandtab ts=4 sw=4 sts=4:
```

Comment:
Unnecessary leftover

Reply:
Why?

Reply:
It's a vim modeline

Reply:
It does not belong in this repo... It's a personal editor preference.

Reply:
Okay, I'll remove them, there are multiple ones here and there.

Reply:
done